sudan_health_nutrition_5.R

```diff
@@ -0,0 +1,28 @@
+# read the data.
+#in so doing, the cmam object can been assigned to values in the Environment
+
+cmam <- read.csv("data/cmam_routine_data.csv")
+
+#the following are the indicators we are interested in to assess reponsiveness of the CMAM project:
+
+# the responsiveness indicators are 5
+#cure rate (cured/discharges)
+#defaulter rate (defaulters/discharged)
+#death rate (dead/discharged)
+#non-response rate (nonresponders/discharged)
+#admission rate (admitted/screened)
+
+#the operational performance indicator is RUTF consumption per month
+
+#adding a column to the dataframe for the calculation of each of these
+
+cmam$curerate <- c(cmam$Cured/cmam$Total.Discharge)
+cmam$defaultrate <- c(cmam$Default/cmam$Total.Discharge)
+cmam$deathrate <- c(cmam$Death/cmam$Total.Discharge)
+cmam$nonresponserate <-c(cmam$Non.Responder/cmam$Total.Discharge)
+cmam$admissionrate <-c(cmam$New.Admissions/cmam$Screening)
```

Comment:
Indicators look good. Interesting concept for admission rate which is not a common performance indicator but your definition for this is interesting. Let's discuss that more next time.

As for the coding itself, a few comments:

* for the variable names, you might consider separating the words with an underscore e.g., instead of `curerate` you can use `cure_rate`. This makes the variable more readable
* for the calculations/operations to get the indicator, no need to put `c()`. This is not really doing anything and is unnecessary. In some cases, this may even cause errors. This should be sufficient:
```R
cmam$cure_rate <- cmam$Cured / cmam$Total.Discharge
```

Reply:
Thanks Ernest, let me get on it. I look forward to our discussion.